versions.txt

```diff
@@ -12,3 +12,14 @@ qemu_lite_version=741f430a960b5b67745670e8270db91aeb083c5f
 mpfr_version=3.1.4
 gmp_version=6.1.0
 mpc_version=1.0.3
+
+# OBS Fedora package versions
+cc_image_fedora_obs_version=16180-29.1
+cc_selinux_fedora_obs_version=0.1-2.1
+linux_container_fedora_obs_version=4.9.35-65.1
```

Comment:
@jcvenegas in this case I think yes we do have specific versions for fedora and ubuntu.